examples/python/Misc/MultiDevice/ptp_frame_sync_minimal.py

```
@@ -0,0 +1,186 @@
+    else:
+        print("Exiting now!")
+        exit(0)
+
+def getSensorName(device: dai.Device, socket: dai.CameraBoardSocket) -> str:
+    sensorName = ""
+    for sckt, sName in device.getCameraSensorNames().items():
+        if sckt == socket:
+            sensorName = sName
+            break
+    if sensorName == "":
+        raise RuntimeError(f"No sensor name found for {socket.name} on {deviceName}")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)